services/paymaster/src/db/config.rs

```diff
@@ -14,17 +14,21 @@ use uuid::Uuid;
 
 use crate::db::pool;
 
-/// Database row where JSON is untyped.
 #[derive(FromRow)]
-struct RawRow {
-    domain_id: Uuid,
+struct DomainConfigRow {
```

Comment:
nit: you don't need to suffix it `Row`. that naming ties it to the name of the table. maybe `DomainConfigMetadata`? i think there's a better way to name this but don't have a suggestion off the top of my head